backend/doc/migrate_from_v1.md

```diff
@@ -59,7 +59,15 @@ After migration, file is accessible in:
 Copy folder `collection-root/agenda/` in `radicale_storage/`.
 
     cp -r export/collection-root /<your>/<tracimv2>/<path>/backend/radicale_storage/
-
+    
+Make sure this file `.Radicale.props` is not exist in this two folder:
```

Comment:
Make sure this file `.Radicale.props` is not exist in this two folder: => Make sure `.Radicale.props` file does not exist in following folders: